data/card_catalogue/unique-1.json

```diff
@@ -379,20 +249,10 @@
       "image": "assets/core/card/illustration/Tripwire Charm.png"
     }
   },
-  {
-    "title": "Counterspell",
-    "upgrade": "",
-    "tier": "red",
-    "details": {
-      "rules": "The effect of the next spell intended for you is applied instead to the caster.",
-      "flavour": "Magic knows no loyalty.",
-      "image": "assets/core/card/illustration/Counterspell.png"
-    }
-  },
   {
     "title": "Large-Caliber Artillery Invocation",
     "upgrade": "",
-    "tier": "red",
+    "tier": "green",
```

Comment:
This card is not strong enough to be red. It's comparable to Anvil Drop, which is green